contracts/mocks/DummyImplementation.sol

```diff
@@ -44,6 +46,12 @@ contract DummyImplementation {
     function reverts() public pure {
         require(false, "DummyImplementation reverted");
     }
+
+    // Use for forcing an unsafe TransparentUpgradeableProxy admin override
+    // solhint-disable-next-line private-vars-leading-underscore
+    function _unsafeOverrideAdmin(address newAdmin) public {
```

Comment:
this is a mock. Do we really want to mess with the linter and add a rule so that this can start with an `_` ?
I'd just call it `unsafeOverrideAdmin`